compiler/sctracker.h

```diff
@@ -19,12 +20,10 @@ typedef struct funcarg_s {
 struct functag_t {
     functag_t()
      : ret_tag(0),
-       argcount(0),
        args()
     {}
     int ret_tag;
-    int argcount;
-    funcarg_t args[SP_MAX_EXEC_PARAMS];
+    PoolList<funcarg_t> args;

```

Comment:
With `-Wall`, gcc complains about signed/unsigned comparison because `argcount` was `int` but `PoolList::length()` is `size_t`.